client/jetpack-cloud/sections/partner-portal/billing-details/index.tsx

```diff
@@ -0,0 +1,151 @@
+									} ) }
+								</span>
+							</div>
+
+							<div className="billing-details__assigned">
+								{ product.counts.assigned }
+								<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+									{ translate( 'Assigned' ) }
+								</span>
+							</div>
+
+							<div className="billing-details__unassigned">
+								{ product.counts.unassigned }
+								<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+									{ translate( 'Unassigned' ) }
+								</span>
+							</div>
+
+							<div className="billing-details__subtotal">
+								{ translate( '%(count)d License', '%(count)d Licenses', {
+									count: product.counts.total,
+									args: { count: product.counts.total },
+								} ) }
+								<span className="billing-details__line-item-meta">
+									{ translate( 'Subtotal: %(subtotal)s', {
+										args: { subtotal: formatCurrency( product.productTotalCost, 'USD' ) },
+									} ) }
+								</span>
+							</div>
+						</div>
+					</Card>
+				) ) }
+
+			{ ! billing.isSuccess && (
+				<Card compact>
+					<div className="billing-details__row">
+						<div className="billing-details__product">
+							<TextPlaceholder />
+							<span className="billing-details__line-item-meta">
+								<TextPlaceholder />
+							</span>
+						</div>
+
+						<div className="billing-details__assigned">
+							<TextPlaceholder />
+							<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+								<TextPlaceholder />
+							</span>
+						</div>
+
+						<div className="billing-details__unassigned">
+							<TextPlaceholder />
+							<span className="billing-details__line-item-meta billing-details__line-item-meta--is-mobile">
+								<TextPlaceholder />
+							</span>
+						</div>
+
+						<div className="billing-details__subtotal">
+							<TextPlaceholder />
+							<span className="billing-details__line-item-meta">
+								<TextPlaceholder />
+							</span>
+						</div>
+					</div>
+				</Card>
+			) }
+
+			<Card compact className="billing-details__footer">
+				<div className="billing-details__row billing-details__row--summary">
+					<span className="billing-details__total-label billing-details__cost-label">
+						{ billing.isSuccess &&
+							translate( 'Cost for {{bold}}%(date)s{{/bold}}', {
+								components: { bold: <strong /> },
+								args: { date: moment( billing.data.date ).format( 'MMMM, YYYY' ) },
+							} ) }
+
+						{ ! billing.isSuccess && <TextPlaceholder /> }
+					</span>
+					<strong className="billing-details__cost-amount">
+						{ billing.isSuccess && formatCurrency( billing.data.costs.total, 'USD' ) }
+
+						{ ! billing.isSuccess && <TextPlaceholder /> }
+					</strong>
+
+					<span className="billing-details__total-label billing-details__line-item-meta">
+						{ translate( 'Assigned licenses:' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'No assigned licenses.' )` *ES Score: 10*